src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,36 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
-
+        int supplyCounter = 0;
+        int buyCounter = 0;
+        Path path = Path.of(fromFileName);
+        Path pathToSave = Path.of(toFileName);
+        try {
+            BufferedReader readedFile = new BufferedReader(new FileReader(String.valueOf(path)));
+            String[] linesArra = Files.readAllLines(path).toArray(new String[0]);
```

Comment:
The variable `linesArra` is misspelled. It should be `linesArray` for better readability and consistency.